mapit_gb/controls/2016-05.py

```diff
@@ -0,0 +1,73 @@
+# A control file for importing May 2016 Boundary-Line. This control file should
+# be okay to run as a first import file also, it uses previous Boundary-Line
+# information if present to fix up issues with the source data.
+
+from mapit.models import Area, Generation
+
+
+def code_version():
+    return 'gss'
+
+
+def check(name, type, country, geometry, ons_code, commit, **args):
+    """Should return True if this area is NEW, False if we should match against ONS code,
+       or an Area to be used as an override instead"""
+
+    if type in ('WAC', 'WAE'):
+        # We might be in the situation where we have bad data, as OS provided
+        # incorrect boundaries from October 2013 to May 2015 inclusive.
+        # October 2015 put the old boundaries in again as new creations.
+        # Let's move the old boundaries to be new, as that's what they are now,
+        # and push back the current boundary generation to cover where they were.
+        try:
```

Comment:
This method looks good to me given the situation you've explained in the comment - have the OS documented that mistake somewhere?